src/main/drivers/itimer.c

```diff
@@ -0,0 +1,17 @@
+
+// Generic interval timer service
```

Comment:
This seems to be a stray file (also, it is picked up by the sanity checks for not including `platform.h`).

Reply:
removed